ig/input/fsh/nbs-obr-segment.fsh

```diff
@@ -27,3 +28,17 @@ Usage: #definition
 * activity = http://terminology.hl7.org/CodeSystem/v3-DataOperation#CREATE
 * agent[+].type = http://terminology.hl7.org/CodeSystem/provenance-participant-type#author
 * agent[=].who.display = "T. R. Johnson"
+
+
+Instance: segment-obr-logical-model-history-create
+InstanceOf: Provenance
+Title: "Removal of OBR-3, OBR-8, OBR-15 and OBR-16 segments"
+Usage: #definition
+* target[+] = Reference(StructureDefinition/segment-obr-logical-model)
+* recorded = "2023-09-07T00:00:00.0000Z"
+* occurredDateTime = "2023-09-07"
+* reason = http://terminology.hl7.org/CodeSystem/v3-ActReason#METAMGT
+* reason.text = "Updated segments to match the ones we initially support"
+* activity = http://terminology.hl7.org/CodeSystem/v3-DataOperation#CREATE
```

Comment:
the `activity` should be `UPDATE`.  double check at the terminology URL for the `v3-DataOperation` code set

Reply:
done.